examples/async_get_job_details.js

```diff
@@ -0,0 +1,21 @@
+const revai = require('revai-node-sdk');
+const fs = require('fs');
+const token = require('../config/config.json').access_token;
+
+(async () => {  
+    // Initialize your client with your revai access token
+    var client = new revai.RevAiApiClient(token);
+
+    // Get account details
+    var account = await client.getAccount();
+    console.log(`Account: ${account.email}`);
+    console.log(`Balance: ${account.balance_seconds} seconds`);
+
+    var filenames = ["example.mp3", "example.mp3"];
```

Comment:
This seems unnecessary since they're both the same. I would see the point if it was easily replaceable but the way this is set up requires that the filename here is relative to the resources folder. I would just get rid of this array and write the filename in the submit

Reply:
An array is used to demonstrate the getListOfJobs functionality. I think for this function to be properly represented it may be better to add another example file rather than submitting only one job.